notebook.ipynb

```diff
@@ -291,47 +231,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "[2023-10-31 14:30:24,515 | fondant.compiler | INFO] Compiling ingestion-pipeline to docker-compose.yml\n",
-      "[2023-10-31 14:30:24,517 | fondant.compiler | INFO] Base path found on local system, setting up ./data-dir as mount volume\n",
-      "[2023-10-31 14:30:24,519 | fondant.pipeline | INFO] Sorting pipeline component graph topologically.\n",
-      "[2023-10-31 14:30:24,526 | fondant.pipeline | INFO] All pipeline component specifications match.\n",
-      "[2023-10-31 14:30:24,527 | fondant.compiler | INFO] Compiling service for load_from_huggingface_hub\n",
-      "[2023-10-31 14:30:24,528 | fondant.compiler | INFO] Compiling service for chunk_text\n",
-      "[2023-10-31 14:30:24,528 | fondant.compiler | INFO] Compiling service for embed_text\n",
-      "[2023-10-31 14:30:24,529 | fondant.compiler | INFO] Compiling service for index_weaviate\n",
-      "[2023-10-31 14:30:24,540 | fondant.compiler | INFO] Successfully compiled to docker-compose.yml\n",
-      " load_from_huggingface_hub Pulling \n",
-      " embed_text Pulling \n",
-      " chunk_text Pulling \n",
-      " index_weaviate Pulling \n",
-      " index_weaviate Error \n",
-      " load_from_huggingface_hub Error \n",
-      "no matching manifest for linux/arm64/v8 in the manifest list entries\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "from fondant.compiler import DockerCompiler\n",
     "from fondant.runner import DockerRunner\n",
     "\n",
-    "DockerCompiler().compile(pipeline)\n",
-    "DockerRunner().run(\"docker-compose.yml\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "If you save the pipeline into a python file you can execute the pipeline using the Fondant cli as well.\n",
+    "compiler = DockerCompiler()\n",
+    "runner = DockerRunner()\n",
     "\n",
-    "> `fondant run local pipeline.py`"
+    "compiler.compile(pipeline=pipeline, output_path = \"docker-compose.yml\")\n",
+    "DockerRunner().run(\"docker-compose.yml\")"
```

Comment:
We already specify `runner = DockerRunner()` above.

For me it's fine to keep this in two lines:
```python
DockerCompiler().compile(pipeline, output_path="docker-compose.yaml")
DockerRunner().run("docker-compose.yaml")
```